recipes/libnftnl/all/conanfile.py

```diff
@@ -0,0 +1,62 @@
+import os
+
+from conan import ConanFile
+from conan.tools.gnu import Autotools, AutotoolsToolchain, PkgConfigDeps
+from conan.tools.files import copy, get, rmdir, rm
+from conan.tools.layout import basic_layout
+from conan.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.53.0"
```

Comment:
```suggestion
required_conan_version = ">=2.0.9"
```
In case `auto_shared_fpic` is used.